stackanetes/k8s_instance.py

```diff
@@ -34,6 +36,8 @@ def __init__(self, service_name, service_dir):
         self.service_type = service_name.split("-")[0]
         self.service_dir = service_dir
         self._load_service_variables()
+        if not check_if_namespace_exist(CONF.stackanetes.namespace):
```

Comment:
```
./kubectl create namespace default
Error from server: namespaces "default" already exists
```

`kubectl create` is already idempotent, we don't really need to check if it does exist. We can just try to create it if non-empty/_default_.